packages/p2p-v2/src/components/Badge/__tests__/Badge.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Badge from '../Badge';
+
+describe('Badge', () => {
+    it('should render the label and statuses correctly', () => {
+        render(<Badge label='Trades' status='100+' variant='general' />);
+        expect(screen.getByText('Trades')).toBeInTheDocument();
+        expect(screen.getByText('100+')).toBeInTheDocument();
```

Comment:
I'm not so sure what else to test here, other than the classNames variant. But those behaviours are controlled by `clsx`, not the component itself